3p/integration.js

```diff
@@ -926,3 +926,8 @@ function lightweightErrorReport(e, isCanary) {
     '&s=' +
     encodeURIComponent(e.stack || '');
 }
+
+window.draw3p();
```

Comment:
1. How do you assure that this code is executed after `<div id="c">` has been parsed?
2. Does `draw3p` ever execute `document.write` today?

Reply:
You are right. I switched it to `document.write` so script is loaded synchronously.

Reply:
Sorry, I meant something different. Does the `draw3p()` contents ever use `document.write()`? It's very strategically placed there inside the specific `<div id="c">` and that makes me think that it's aimed to enable `document.write`.

Reply:
Yes, `document.write` is being used. I changed this also to load the js w/ `document.write` so that `document.write` is still available during `draw3p()`.